runtime/lua/vim/treesitter/query.lua

```diff
@@ -184,6 +184,76 @@ local function read_query_files(filenames)
   return table.concat(contents, '')
 end
 
+---@param node TSNode
+---@param bufnr integer
+---@return integer, integer, integer, integer
+local function node_trim_trailing_blank_lines(node, bufnr)
```

Comment:
"blank" includes lines with whitespace. Based on your comment, it sounds like it only empty lines are trimmed?
```suggestion
local function node_trim_trailing_empty_lines(node, bufnr)
```